city_scrapers/spiders/chi_ssa_51.py

```diff
@@ -0,0 +1,75 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa51Spider(CityScrapersSpider):
+    name = "chi_ssa_51"
+    agency = "Chicago Special Service Area #51 Chatham"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cbatechworks.org"]
+    start_urls = ["http://www.cbatechworks.org/"]
+    location = {"address": "806 East 78th Street, Chicago IL 60619", "name": "QBG Foundation"}
+    last_parsed_date = None
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css("div#element106 font"):
+            """
+            The date and times contained within sibling divs that are identicals, so we have to
+            continue the loop and only create the meeting until both date and times have been
+            parsed.
+            """
+            if not self.last_parsed_date:
+                self._parse_date(item)
+                continue
+            else:
+                start_and_end = self._parse_time(item)
+                if not start_and_end:
+                    continue
+                start = self.last_parsed_date + " " + start_and_end[0].strip()
+                start = datetime.strptime(start, "%B %d, %Y %I:%M%p")
+                end = self.last_parsed_date + " " + start_and_end[1].strip()
+                end = datetime.strptime(end, "%B %d, %Y %I:%M%p")
+                self.last_parsed_date = None
+
+            meeting = Meeting(
+                title="Commission",
+                description="",
+                classification=COMMISSION,
+                start=start,
+                end=end,
+                all_day=False,
+                time_notes="",
+                location=self.location,
```

Comment:
It would be good to add a basic `_validate_location` function here like we have in [`chi_ssa_34`](https://github.com/City-Bureau/city-scrapers/blob/c23f1fd17b1ced1c01dff295c8cdf1ca191f0244/city_scrapers/spiders/chi_ssa_34.py#L58). Since this SSA clearly includes the address which doesn't change between meetings, we try to do a sanity check to make sure the address is the same. Otherwise we want to throw an error so that we get a notification from Sentry